examples/dreambooth_custom_character/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating a Cute Custom Character with Stable Diffusion and Dreambooth
+
+## Introduction
+[Stable Diffusion](https://github.com/CompVis/stable-diffusion) is an incredible open-source tool for fast, effective generation of novel images across a wide variety of domains. Despite its power and convenience, it can struggle to create consistent, identifiable characters in specific styles.
+
+While improvements in prompting can mitigate this to some extent, it is almost always a better bet to fine-tune the model with your own data. Here are a few reasons why: 
```

Comment:
Let's move this below our 404 ox, and add a title "Why fine-tune in the first place?" or something